specification/marketplacecatalog/data-plane/Search/stable/2022-02-02/search.json

```diff
@@ -0,0 +1,1065 @@
+{
+  "x-generator": "NSwag v13.15.10.0 (NJsonSchema v10.6.10.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Search API",
+    "version": "2022-02-02"
+  },
+  "host": "catalogapi.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "text/plain",
+    "application/json",
+    "text/json"
+  ],
```

Comment:
If most of your operations return JSON -- which they should -- then you should only have "application/json" here and then override in specific methods where necessary. Declaring all the content types globally will produce very poor generated client libraries.

Reply:
done